cmd/promtool/main.go

```diff
@@ -103,6 +103,29 @@ func main() {
 		"The unit test file.",
 	).Required().ExistingFiles()
 
+	defaultDBPath := filepath.Join("benchout", "storage")
```

Comment:
now that tsdb has been moved recently all default need to be revised. Could you test these and update accordingly.

Reply:
Is there any other path I need to update besides the tsdb path?

Reply:
just run the command locally and you will find out if any returns an error.

Reply:
I have already tested in my local env and it works well. Can you also check it?